test/parallel/test-stream-readable-unpipe-resume.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const common = require('../common');
+
+const stream = require('stream')
+
+const fs = require('fs');
+const readStream = fs.createReadStream('out/Release/node')
```

Comment:
Needs a largish file. Is this fine?

Reply:
I think using the Node.js binary is fine; but please refer to it with `process.execPath` :)